src/Tezos/Address.cpp

```diff
@@ -24,12 +24,18 @@ bool Address::isValid(const std::string& string) {
     }
 
     // verify prefix
-    std::array<byte, 3> prefix{6, 161, 159};
-    if (!std::equal(prefix.begin(), prefix.end(), decoded.begin())) {
-        return false;
+    std::array<byte, 3> tz1Prefix{6, 161, 159};
```

Comment:
can we make them constants? I saw `prefix{6, 161, 159}` in `Tezos::Address` too

Reply:
Sure. 

I refactored forging.cpp as well to not use these constants so we don't need to extern. 

There are forge_tzX test cases in AddressTests.cpp which validate this code works.